src/app/features/login/login.reducers.ts

```diff
@@ -0,0 +1,16 @@
+import {UserModel} from "../../shared/models/user.model";
+import {createReducer, on} from "@ngrx/store";
+import * as LoginPageActions from "./login.actions";
+import {AuthorizedUserModel} from "../../shared/models/authorized-user.model";
+
+export const initialUser: AuthorizedUserModel = {
+  user: {} as UserModel,
+  token: ''
+}
+
+export const loginReducer = createReducer(
+  initialUser,
+  on(LoginPageActions.register, (state) => ({
```

Comment:
it would also be nice (not needed) if you could add loading and error states, for example you would have 3 actions then, register, registerSucess, registerFail (or however you want to call the success and fail up to you) that way once it's successful you can update the user state, loadign state to false and error to null, if it failed then user state stays the same , error state updated, loading false, and on pressing register you would only set loading to true, it can introduce loading spinners or skeletons and error/success messages for the application which greatly improves user experience